frontend/src/features/properties/map/MapView.tsx

```diff
@@ -34,16 +35,16 @@ const MapView: React.FC<MapViewProps> = (props: MapViewProps) => {
   const { sidebarComponent, showSideBar } = useMapSideBarQueryParams(mapInstance);
 
   const onMarkerClicked = (property: IProperty) => {
-    const parsedPid = pidParser(property.pid);
-    history.push(`/mapview/property/${parsedPid}`);
+    history.push(`/mapview/property/${property.id}?pid=${property.pid}`);
   };
 
   const onPropertyViewClicked = (pid?: string | null) => {
     if (pid !== undefined && pid !== null) {
       const parsedPid = pidParser(pid);
-      history.push(`/mapview/property/${parsedPid}`);
+      history.push(`/mapview/non-inventory-property/${parsedPid}`);
```

Comment:
In this case the id of the property would not be the pid. Thoughts on using "?pid=" on both?

like:
/mapview/property?id=${propertyId}&pid={propertyPid}
/mapview/property?pid=${propertyPid}

Reply:
Hmm, I thought this would be ok since a non-inventory property would never have an id so the main identifier would be a pid, but the above also works.